.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
Please share some context here why you required to specify the distribution?

Reply:
The apt packages `libblas3gf`, `liblapack3gf`  cannot be installed on ubuntu `version >14` but by default, Travis uses ubuntu `16`. so I have to pin ubuntu `14` code name `trusty`.

For further details checkout this [thread](https://askubuntu.com/questions/989778/e-unable-to-locate-package-libblas3gf)